src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -93,7 +93,7 @@ You can also apply this configuration when unsetting a user at logout:
 To learn what useful information can be associated with scopes see
 [the context documentation](../context/).
 
-<PlatformSection notSupported={["android"]}>
+<PlatformSection>
```

Comment:
Not exactly, it shows the same doc (like all SDKs), but only the code example for the callback (like dotnet). Java on the other hand has code examples for both the old and the new version.

However, as mentioned above, we do have a discrepancy in the SDKs anyways. Some use an enclosure like the original java withScope and some use a callback-based approach on the capture methods directly.

In my opinion we should have 2 Sections for LocalScope something like
- Local Scope
- Local Scope Callback

and show them for the relevant SDKs.

Because the current doc doesn't really make sense for all the Callback based SDKs. For Java we could show both sections in the meantime.

Reply:
For Java we could also show an alert that the `withScope` variant doesn't play well with `globalHubMode`

Reply:
Sounds good, I'd move the current docs into a subheading and have a sibling subheading for the `capture` overload with callback param.

maybe like this:

```
## Local Scopes
### Using `withScope`
### Using Scope Callback Parameter
```


Warning sounds good as well.